pipeline_steps/tempest.groovy

```diff
@@ -29,28 +23,25 @@ def tempest_run(wrapper="") {
 /* if tempest install fails, don't bother trying to run or collect test results
  * however if running fails, we should still collect the failed results
  */
-def tempest(deploy_vm=null){
-  if (deploy_vm != null) {
-    wrapper = "sudo ssh -T -oStrictHostKeyChecking=no ${deploy_vm}"
-    copy_cmd = "scp -o StrictHostKeyChecking=no -p  -r \$(${wrapper} \"cd /opt/rpc-openstack/openstack-ansible/playbooks && ansible utility[0] -m command -a 'echo {{ physical_host }}' | grep -Ev 'Variable files|utility'\"):"
-  } else{
-    wrapper = ""
-    copy_cmd = "cp -p "
-  }
+def tempest(){
+  find_tempest_results_cmd = "find /var/lib/lxc/*utility*/ -type f -name \"tempest_results.xml\""
+  tempest_server = "\$(cd /opt/rpc-openstack/openstack-ansible/playbooks && ansible utility[0] -m command -a 'echo {{ physical_host }}' | grep -Ev 'Variable files|utility')"
+  copy_cmd = "scp -o StrictHostKeyChecking=no -p  -r ${tempest_server}:"
+  tempest_results_location = "\$(ssh -o StrictHostKeyChecking=no ${tempest_server} \"${find_tempest_results_cmd}\")"
```

Comment:
Updated commit to remove the `tempest_results_location` variable and put that in the script for more clarity on what exactly is going on. I think breaking it down any further is really going to get sloppy and hard to track subshells and groovy escaping.